ClemBot.Bot/bot/cogs/emote_board_cog.py

```diff
@@ -0,0 +1,430 @@
+
+        embed = discord.Embed(title=":placard: Emote Boards", color=Colors.ClemsonOrange)
+        embed.description = "The allowance of bot posts for your board has been updated."
+        embed.add_field(name="Name", value=board.name)
+        embed.add_field(name="Emote", value=board.emote)
+        embed.add_field(name="Allow Bot Posts", value=str(bots))
+        embed.set_footer(text=str(ctx.author), icon_url=ctx.author.display_avatar.url)
+        await ctx.send(embed=embed)
+
+    @set_group.command(pass_context=True, name="emote", aliases=["emoji"])
+    @ext.required_claims(Claims.manage_emote_boards)
+    @ext.long_help("Sets the emote that belongs to the given emote board.")
+    @ext.short_help("Sets the emote for a board.")
+    @ext.example(["board set emote starboard :star:", "board set emote starboard :a_custom_emote:"])
+    async def set_emote(
+        self,
+        ctx: ext.ClemBotCtx,
+        board_name: str,
+        emote: Annotated[discord.Emoji | str, EmoteConverter],
+    ) -> None:
+        if not (board := await self._get_board(board_name, ctx)):
+            return
+
+        if not self._is_emoji(emote):
+            await self._error_embed(ctx, f"`{emote}` is not a valid emote or emoji.")
+            return
+
+        if await self._get_board(emote, ctx, False):
+            await self._error_embed(
+                ctx, f"An emote board with the given emote {emote} already exists."
+            )
+            return
+
+        board.emote = emote if isinstance(emote, str) else str(emote)
+        await self.bot.emote_board_route.edit_emote_board(ctx.guild, board, raise_on_error=True)
+
+        embed = discord.Embed(title=":placard: Emote Boards", color=Colors.ClemsonOrange)
+        embed.description = "The emote for your board has been updated."
+        embed.add_field(name="Name", value=board_name)
+        embed.add_field(name="Emote", value=emote)
+        embed.set_footer(text=str(ctx.author), icon_url=ctx.author.display_avatar.url)
+        await ctx.send(embed=embed)
+
+    @emoteboard.group(name="channel", aliases=["channels"])
+    @ext.long_help("Add or remove a channel for ClemBot to post to for a specific emote board.")
+    @ext.short_help("Add or remove a channel from an emote board.")
+    @ext.example("help emoteboard channel")
+    async def channel_group(self, ctx: ext.ClemBotCtx) -> None:
+        return
+
+    @channel_group.command(pass_context=True, name="add")
+    @ext.required_claims(Claims.manage_emote_boards)
+    @ext.long_help("Add a channel for ClemBot to post messages to for the given emote board.")
+    @ext.short_help("Add a channel for an emote board.")
+    @ext.example(
+        [
+            "board channel add :star: #my-cool-channel",
+            "board channel add starboard #another-channel",
+        ]
+    )
+    async def channel_add(
+        self, ctx: ext.ClemBotCtx, channel: discord.TextChannel, emoteboard: EMOTEBOARD_TYPE
```

Comment:
Yeah it felt kinda weird to have it be like `!board channel add ⭐ #my-channel` instead of `!board channel add #my-channel ⭐`. If you prefer one over the other just lmk.

Reply:
I don't care which way you do it, i was just saying to flipped the command order from what your wrote in the docs example